tests/unit/oidc/test_tasks.py

```diff
@@ -95,3 +98,62 @@ def test_compute_oidc_metrics(db_request, metrics):
             "warehouse.oidc.publishers", 4, tags=["publisher:github_oidc_publishers"]
         ),
     ]
+
+
+def test_delete_expired_oidc_macaroons(db_request, macaroon_service, metrics):
+    # We'll create 3 macaroons:
+    # - An OIDC macaroon with creation time of 1 day ago
+    # - An OIDC macaroon with creation time now
+    # - A non-OIDC macaroon with creation time of 1 day ago
+    # The task should only delete the first one
```

Comment:
Let's add another OIDC macaroon here, one that's expired by has been expired for `< 1` day 🙂